src/lm_buddy/jobs/configs/ragas.py

```diff
@@ -0,0 +1,60 @@
+from pydantic import Field, field_validator
+
+from lm_buddy.integrations.huggingface import AutoModelConfig
+from lm_buddy.integrations.huggingface.dataset_config import TextDatasetConfig
+from lm_buddy.integrations.vllm import VLLMCompletionsConfig
+from lm_buddy.integrations.wandb import WandbRunConfig
+from lm_buddy.types import BaseLMBuddyConfig
+
+
+class RagasEvaluationConfig(BaseLMBuddyConfig):
+    """Parameters specifically required for RAGAs Evaluation"""
+
+    metrics: list[str] | None = [
```

Comment:
If you want to ensure the values passed here are correct, Pydantic validation can help.

```
from typing import Literal

RagasEvaluationMetric = Literal["faithfulness", "answer_relevancy", ...]

class RagasEvaluationConfig(BaseLMBuddyConfig):
    metrics: list[RagasEvaluationMetric]
```

Also, [Python has funky behavior with default values for mutable data structures.](https://docs.python-guide.org/writing/gotchas/#:~:text=Mutable%20Default%20Arguments%C2%B6). If you want to specify a default here, you should use the `Field(..., default_factory=list())` function from Pydantic.